cmd/utils/flags.go

```diff
@@ -1497,6 +1509,14 @@ func SetNodeConfig(ctx *cli.Context, cfg *node.Config) {
 	if ctx.IsSet(InsecureUnlockAllowedFlag.Name) {
 		cfg.InsecureUnlockAllowed = ctx.Bool(InsecureUnlockAllowedFlag.Name)
 	}
+	if ctx.IsSet(BackingDBFlag.Name) {
+		backingDB := ctx.String(BackingDBFlag.Name)
+		if backingDB != "leveldb" && backingDB != "pebble" {
+			Fatalf("invalid choice for backing db: %s", backingDB)
```

Comment:
Please change to `Invalid choice for backing db: '%s', allowed: 'leveldb' or 'pebble'`